### ISSUE TITLE
[BUG] OpenVPN 2.6.9+ can't be installed/updated on Windows 11

### ISSUE THREAD

**Issue by Athar42** [OpenVPN/openvpn-gui#692]
Hi,

I struggle to update my openVPN client on my PC.

While trying to update from 2.6.1 to 2.6.9/2.6.10/2.6.11, I was unable to do so, always getting an error during the setup process :

```
Product: OpenVPN 2.6.9-I001 amd64 -- Error 1720. There is a problem with this Windows Installer package. A script required for this install to complete could not be run. Contact your support personnel or package vendor.  Custom action SetACL script error -2147467259, Msi API Error: Property,Name Line 176, Column 9,  
```

```
Product: OpenVPN 2.6.11-I001 amd64 -- Error 1720. There is a problem with this Windows Installer package. A script required for this install to complete could not be run. Contact your support personnel or package vendor.  Custom action SetACL script error -2147467259, Msi API Error: Property,Name Line 176, Column 9,  
```

This issue seems related to some new security added to the installer, which set the access permissions to folders/services and failed on my PC.

Now, I updated until the failure, all the way to 2.6.8, and the first failure start at update of 2.6.9.

OS : Windows 11 - 23H2 - Build 22631.3737
OS Lang : French

**Comment by lstipakov** [comment]
Duplicate of https://github.com/OpenVPN/openvpn-build/issues/671

**Comment by lstipakov** [comment]
@Athar42 could you please test [this](https://github.com/OpenVPN/openvpn-build/actions/runs/9660875017/artifacts/1635485569) installer?

**Comment by Athar42** [comment]
When I try to update one time, I got this at first :

![image](https://github.com/OpenVPN/openvpn-gui/assets/44590042/16286639-db12-4caf-8f4c-aac32e2a8264)

I then rerun the setup through a command line as Administrator and it pass without error.

I used the "custom" setup (which doesn't display the "UAC shield" on the button), while doing so, it didn't request to elevate the permission (anyways, it's disabled (UAC) on my PC, it didn't request for it, and I'm an "Administrator" account too).

Not a big deal, just something odd that it didn't elevate its permissions by itself.

But regarding the initial issue reported, this is fixed :)
